nimare/tests/test_meta_mkda.py

```diff
@@ -53,7 +53,16 @@ def test_mkda_density(testdata_cbma):
     assert isinstance(cres, nimare.results.MetaResult)
 
 
-def test_mkda_chi2_fdr(testdata_cbma):
+def test_MKDADensity_low_memory(testdata_cbma):
+    """Smoke test for MKDADensity with low_memory option."""
+    meta = MKDADensity(null_method="empirical", n_iters=10, low_memory=True)
+    res = meta.fit(testdata_cbma)
+    assert meta.low_memory
+    assert not meta.kernel_transformer.low_memory
```

Comment:
is there a use case where we would want the estimator and kernel to have different settings? Or should low_memory be set in the kernel what the setting is in the estimator?

Reply:
I'm not sure. I _do_ want to make sure there isn't unintentional bleedover of parameters, but it does seem reasonable to do it intentionally. We'll want to log something about overwriting the kernel transformer's parameter though.

Reply:
I think I want to handle this in a separate PR. It should be fairly easy to implement, but since @62442katieb is waiting on these changes for the metaCluster workflow I'd like to get it merged sooner rather than later.

Reply:
sounds good to me